<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>657</width>
    <height>446</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widgetResult" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="lineEditTips">
        <property name="readOnly">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonBack">
        <property name="text">
         <string>退格</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditResultName">
        <property name="maximumSize">
         <size>
          <width>220</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="readOnly">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonClear">
        <property name="text">
         <string>清空</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widgetParent" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="pushButtonFuqin">
        <property name="text">
         <string>父亲</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonMuqin">
        <property name="text">
         <string>母亲</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widgetFamily" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="pushButtonLaogong">
        <property name="text">
         <string>老公</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonLaopo">
        <property name="text">
         <string>老婆</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonErzi">
        <property name="text">
         <string>儿子</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonNver">
        <property name="text">
         <string>女儿</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widgetRelation" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QPushButton" name="pushButtonGege">
        <property name="text">
         <string>哥哥</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonDidi">
        <property name="text">
         <string>弟弟</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonJiejie">
        <property name="text">
         <string>姐姐</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonMeimei">
        <property name="text">
         <string>妹妹</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widgetVirtualRelation" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QPushButton" name="pushButtonYifu">
        <property name="text">
         <string>义父</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonYimu">
        <property name="text">
         <string>义母</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonYizi">
        <property name="text">
         <string>义子</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonYinv">
        <property name="text">
         <string>义女</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
